### ISSUE TITLE
Generating code for SSP

### ISSUE THREAD

**Issue by njericha** [JacquesCarette/literate-scientific-software#351]
```
stack exec ssp
ssp.EXE: Case should be dealt with separately
CallStack (from HasCallStack):
  error, called at .\Language\Drasil\Code\Imperative\Import.hs:349:25 in drasil-0.0.8-6NGR5FopxtSDYmIYT1BGPX:Language.Drasil.Code.Imperative.Import
make.exe": *** [ssp] Error 1
```
I got this error while running `make ssp` and I followed the format of glassBR.

**Comment by palmerst** [comment]
This is due to how I handled `Case` in `Expr` in the generator.  I will fix this shortly.

**Comment by palmerst** [comment]
Fixed in  18ea21e
